demo/demo2.html

```diff
@@ -51,13 +51,13 @@
 </head>
 <body unresolved>
 
-  <app-drawer-layout>
+  <app-drawer-layout fullbleed>
 
     <app-drawer>
       <app-toolbar>App name</app-toolbar>
     </app-drawer>
 
-    <app-header-layout>
+    <app-header-layout fullbleed>
```

Comment:
Why are changing all the demos to use `fullbleed`?  Should we create new demos for showing `fullbleed` usage?